test/functional/apps/dashboard_elements/input_control_vis/input_control_options.ts

```diff
@@ -17,19 +17,17 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const inspector = getService('inspector');
   const find = getService('find');
   const comboBox = getService('comboBox');
-
   const FIELD_NAME = 'machine.os.raw';
 
+  const from = 'Jan 1, 2017 @ 00:00:00.000';
+  const to = 'Jan 1, 2017 @ 00:00:00.000';
+
   describe('input control options', () => {
     before(async () => {
       await PageObjects.visualize.initTests();
       await PageObjects.visualize.navigateToNewVisualization();
       await PageObjects.visualize.clickInputControlVis();
-      // set time range to time with no documents - input controls do not use time filter be default
-      await PageObjects.timePicker.setAbsoluteRange(
-        'Jan 1, 2017 @ 00:00:00.000',
-        'Jan 1, 2017 @ 00:00:00.000'
-      );
+      await PageObjects.common.setTime({ from, to });
```

Comment:
Setting the default time this way doesn't change the timepicker after the test has already navigated to a page.  It needs to be done before navigating.  The test may pass, but not in the same was as it currently is in main.

Reply:
have moved the timepicker before navigating to the page.